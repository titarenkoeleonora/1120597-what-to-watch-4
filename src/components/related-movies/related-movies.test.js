import React from "react";
import renderer from "react-test-renderer";
import RelatedMovies from "./related-movies";

const testMovie = {
  title: `Movie title`,
  genre: `Genre`,
  date: 2020,
  poster: `image`,
  bgImage: `bg-image`,
  src: ``,
};

const testMovies = [
  {
    id: `0`,
    title: `movie title`,
    poster: `image`,
    src: ``,
  },
  {
    id: `1`,
    title: `movie title`,
    poster: `image`,
    src: ``,
  },
  {
    id: `2`,
    title: `movie title`,
    poster: `image`,
    src: ``,
  },
  {
    id: `3`,
    title: `movie title`,
    poster: `image`,
    src: ``,
  },
  {
    id: `4`,
    title: `movie title`,
    poster: `image`,
    src: ``,
  },
  {
    id: `5`,
    title: `movie title`,
    poster: `image`,
    src: ``,
  },
  {
    id: `6`,
    title: `movie title`,
    poster: `image`,
    src: ``,
  },
  {
    id: `7`,
    title: `movie title`,
    poster: `image`,
    src: ``,
  }
];

it(`RelatedMovies should render correctly`, () => {
  const tree = renderer
      .create(<RelatedMovies
        currentMovie={testMovie}
        movies={testMovies}
        onMovieCardClick={() => {}}
      />).toJSON();

  expect(tree).toMatchSnapshot();
});
