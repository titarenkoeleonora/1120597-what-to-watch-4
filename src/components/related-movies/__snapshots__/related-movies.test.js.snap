// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RelatedMovies should render correctly 1`] = `
<div
  className="catalog__movies-list"
/>
`;
